uspv/init.go

```diff
@@ -137,6 +173,15 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return err
 	}
 	s.WBytes += uint64(n)
+	return nil
+}
+
+// Connect dials out and connects to full nodes.
+func (s *SPVCon) Connect(remoteNode string) error {
```

Comment:
I think this should be organized so that more is going on in Connect().  Connect can call all the sub-functions directly